### DISCUSSION TITLE
Nextjs Example

### DISCUSSION THREAD

**Discussion by noteverseio** [dream-num/univer#1449] [Q&A | 常见问题]
Dear authors, thanks for such a great project, I am wondering if we have a nextjs example. I was trying to use this project in a nextjs project but failed to bootstrap univer/sheet due to some syntax errors, anyone had tried nextjs with univer/sheet before?

**Comment by jikkai** [comment]
Thank you for your interest in our project! 

We have indeed created a demo that integrates Univer with Next.js, which you can find at [sheets-nextjs-demo](https://github.com/awesome-univer/sheets-nextjs-demo).

However, please be aware that this demo is currently not functional due to an issue outlined in this pull request: [fix(engine-formula): ensure super is called at the beginning of the constructor](https://github.com/dream-num/univer/pull/1452). The demo will be operational once version 0.1.2 is released.

In the meantime, if you're eager to get started, you might want to try our nightly builds following the guidance provided in our documentation here: [Univer Docs - Other Releases](https://univer.ai/guides/release/).

**Comment by noteverseio** [comment]
Thank you so much, this is the exact error I have seen, will try the alpha release;)

**Comment by thanh-withcoverage** [comment]
Hi @jikkai , Do you have plan to support NextJS anytime soon? Your demo at https://github.com/awesome-univer/sheets-nextjs-demo fails with latest version 0.1.9 due to using `fs` 
![image](https://github.com/dream-num/univer/assets/165621204/20cb65b7-cd13-4af1-99d3-11adf0afb96c)

**Reply by thanh-withcoverage** [reply]
I was able to setup everything working with version 0.1.7. However, I have another issue with library `@wendellhu/redi`. Most of the code is similar to your demo.
![image](https://github.com/dream-num/univer/assets/165621204/e63e652c-8617-4a22-860b-f5584b5ef0dc)
